dll/win32/comctl32/combo.c

```diff
@@ -1610,8 +1610,9 @@ static void COMBO_LButtonDown( LPHEADCOMBO lphc, LPARAM lParam )
        else
        {
 	   /* drop down the listbox and start tracking */
-
+#ifndef __REACTOS__
            lphc->wState |= CBF_CAPTURE;
+#endif
```

Comment:
I would like to understand the following.
This code path still starts the capture (see below); however now the flag that says "capture" is not set, so there's some sort of "discrepancy" between the actual state and the translation in terms of flags (in `wState`).

Is the capture started by `SetCapture()` correctly reset later, whatever the value of `wState` ?

Reply:
@HBelusca, Please review after my recent changes and see if the wState doesn't track the SetCapture() now while still retaining the improvements to the drop-down listbox. Thanks.